.github/workflows/promote_charm_haproxy.yaml

```
@@ -27,4 +27,5 @@ jobs:
       base-channel: 24.04
       base-name: ubuntu
       base-architecture: amd64
+      tag-prefix: haproxy-operator
```

Maybe we can be less verbose? No strong opinion, just and idea.
```suggestion
      tag-prefix: haproxy
```

---

That seems fine to me, thanks!